frontend/src/Settings/Indexers/Options/IndexerOptions.js

```diff
@@ -85,7 +85,7 @@ function IndexerOptions(props) {
                 type={inputTypes.CHECK}
                 name="preferIndexerFlags"
                 helpText={translate('PreferIndexerFlagsHelpText')}
-                helpLink="https://wiki.servarr.com/Definitions#Indexer_Flags"
+                helpLink="https://wiki.servarr.com/radarr/settings#options"
```

Comment:
```suggestion
                helpLink="https://wiki.servarr.com/radarr/settings#indexer-flags"
```
just the quick link fixup :)

Reply:
I promise I can read 😓